admob/integration_test/build.gradle

```diff
@@ -73,6 +77,10 @@ android {
 apply from: "$gradle.firebase_cpp_sdk_dir/Android/firebase_dependencies.gradle"
 firebaseCpp.dependencies {
   admob
+  messaging
```

Comment:
I'm cool with leaving these, but we should probably put a comment explaining why

Reply:
Oh no I should remove that, it was just for testing.